README.md

```diff
@@ -53,8 +53,8 @@ Current mode version: *2020.3.8_parsing*
 |---|---|
 |author|0.9053|
 |title|0.8607|
-|year|0.0.8639|
-|o|0.0.9340|
```

Comment:
The other thing is that these values match the values in the 2020.3.7 parsing table not the 2020.3.8 table
https://docs.google.com/spreadsheets/d/1gu6jJ83Ad15VztmB2aCGTgVsQgaudr6rwbU6RO0YV-I/edit#gid=1060813097
There is also a discrepancy for the results here for the `2020.4.5_multitask` model and the results in the excel

Reply:
ahh could be that I just didn't update them...